addon/webextension/background/startBackground.js

```diff
@@ -0,0 +1,94 @@
+/* globals browser, main, communication */
+/* This file handles browser.browser.onClicked, and loads the rest of the background
```

Comment:
Maybe also mention that the context menu click can initiate loading as well, just for consistency